package.json

```diff
@@ -147,7 +144,8 @@
         "webpack": "^5.72.1",
         "webpack-cli": "^4.9.2",
         "webpack-dev-server": "^4.9.0",
-        "yaml-flat-loader": "^1.0.1"
+        "yaml-flat-loader": "^1.0.1",
```

Comment:
```suggestion
```

Reply:
I removed unnecessary script from package.json.
[refactor: remove yarn-flat-loader](https://github.com/reearth/reearth-web/pull/240/commits/923c78453d166482d5439369804b452d86905f5f)
[refactor: remove unnecessary script](https://github.com/reearth/reearth-web/pull/240/commits/6351eed1bc1425350c0b6c91b2d092cb1b86bf00)

Reply:
One of your commits after the `remove yarn-flat-loader` commit brought it back. Please delete it again